site/kubernetes/operator/using-operator.md

```diff
@@ -1034,3 +1037,23 @@ or use
 <pre class="lang-bash">
 kubectl delete -f INSTANCE.yaml
 </pre>
+
+## <a id='pause' class='anchor' href='#pause'>Pause Reconciliation for a RabbitMQCluster</a>
+
+If you wish to pause reconciliation for a RabbitMQ instance to stop the cluster operator updating and watching the instance. You can set a special label on your RabbitmqCluster.
```

Comment:
We generally do not use "tutorial-oriented" language such as "you can X" in our guides. This would sounds a bit like this if this was in an existing guide:

"It is possible to pause reconciliation for a RabbitMQ instance: this will prevent the cluster operator from watching and updating the instance. To do so, set a special label on your RabbitmqCluster…"

Reply:
Updated!